ci/jenkins/test.sh

```diff
@@ -558,7 +558,7 @@ function run_e2e {
     # HACK: see https://github.com/antrea-io/antrea/issues/2292
     go mod edit -replace github.com/moby/spdystream=github.com/antoninbas/spdystream@v0.2.1 && go mod tidy
     if [[ $TESTBED_TYPE == "flexible-ipam" ]]; then
-        go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider remote -timeout=100m --prometheus --antrea-ipam
+        go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider remote -timeout=120m --prometheus --antrea-ipam
```

Comment:
Do we need 20 more mins? I am worried about the test time. @tnqn

Reply:
The additional 20mins are for DualStack e2e, since it will run both IPv4 and IPv6 cases.
Do you think we can skip some cases for DualStack e2e to reduce the time cost?

Reply:
I think we could add condition similar to `skipIfNotRequired(t, "mode-irrelevant")` to skip ipam irrelevant tests, for example `skipIfNotRequired(t, "ipam-irrelevant")`, tests like `TestAntreaPolicy`, `TestSecurity`, `TestAntctl` may apply.

Reply:
Since we scaled up the flexible-ipam-e2e testbed cluster and this change only extend 20mins for flexible-ipam-e2e, the time consumed by e2e cases is acceptable, and we can focus on new feature in this PR.